packages/lambda_streams/lambda_streams.0.1.1/opam

```diff
@@ -0,0 +1,42 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+version: "0.1.1"
+synopsis: "Lambda-based streaming library"
+description: "A simple and principled streaming library based on lambdas"
+maintainer: ["me@risto.codes"]
+authors: ["Risto Stevcev"]
+license: "BSD-3-Clause"
+homepage: "https://github.com/Risto-Stevcev/lambda-streams"
+doc: "https://risto-stevcev.github.io/lambda-streams"
+bug-reports: "https://github.com/Risto-Stevcev/lambda-streams/issues"
+depends: [
+  "ocaml" {>= "4.06.1" & <= "4.10.0"}
```

Comment:
Is there a reason to stop at OCaml 4.10.0 precisely? If you know you rely on something that will change in OCaml 4.11 it is a reasonable thing to add an upper-bound constraint on `< "4.11"`, but I find very surprising that this package wouldn't be compatible with OCaml 4.10.1 for instance. If this is not the case, would it be ok to remove the upper-bound constraint entirely?

Reply:
What would happen if ocaml >= 4.11 introduces a change that would break this library though?